ocaml/idl/datamodel.ml

```diff
@@ -7069,6 +7069,7 @@ let pool_set_vswitch_controller = call
       Published, rel_midnight_ride, "Set the IP address of the vswitch controller.";
       Extended, rel_cowley, "Allow to be set to the empty string (no controller is used)."]
     ~name:"set_vswitch_controller"
+    ~internal_deprecated_since:rel_falcon
```

Comment:
Add a `Deprecated` entry to the `~lifecycle` above, with explanation string, rather than using `~internal_deprecated_since`.